gslab_scons/size_warning.py

```diff
@@ -64,15 +64,14 @@ def issue_size_warnings(look_in = ['source', 'raw', 'release'],
 
         # if there are large versioned files not tracked by git-lfs, warned the user and perform tracking if the user agrees 
         if new_add_list:
-            print "The following files are versioned large files that " + \
-            "are not tracked by git-lfs (recommand using git-lfs to track them): " 
+            print _red_and_bold("The following files are versioned large files that " + \
+            "are not tracked by git-lfs (recommend using git-lfs to track them): ") 
```

Comment:
I think we want to make the style consistent with the other warnings, like [this](https://github.com/gslab-econ/gslab_python/blob/f15f092b4633409c2ba2eac13dab5edc2d0edae1/gslab_scons/size_warning.py#L58). Only the word `Warnings` is red and bold.